src/util/validation.js

```diff
@@ -98,13 +98,19 @@ const objectIdParamsSchema = customJoi.object().keys({
   id: customJoi.string().required()
 }).required()
 
-function getRangeFilter (joiType) {
+function getRangeFilter (joiType, forbidSameBoundaryType = false) {
```

Comment:
I suppose you will remove this parameter and make `true` the default behavior when migrating current list endpoints.

Reply:
Exactly